docs/useListController.md

```diff
@@ -85,6 +88,28 @@ const MyList = ({
 };
 ```
 
+## `storeKey`
+
+To display multiple lists of the same resource and keep distinct store states for each of them (filters, sorting and pagination), specify unique keys with the `storeKey` property.
```

Comment:
Maybe mention somewhere in this section that the default value is `${resource}.listParams`